cps/CPXX.md

```diff
@@ -0,0 +1,76 @@
+**Author(s):** Governance and Coordination Group (ImdioR, Orhan)

```

Comment:
I would add a title here:

`# CP6: Implement a block reward for Centrifuge Parachain`

Reply:
You should rename this file to CP6.md you can do this in the editor